pkg/printers/internalversion/describe.go

```diff
@@ -3392,6 +3392,7 @@ func describePodSecurityPolicy(psp *extensions.PodSecurityPolicy) (string, error
 		w.Write(LEVEL_0, "\nSettings:\n")
 
 		w.Write(LEVEL_1, "Allow Privileged:\t%t\n", psp.Spec.Privileged)
+		w.Write(LEVEL_1, "Allow Privilege Escalation:\t%v\n", psp.Spec.AllowPrivilegeEscalation)
```

Comment:
@liggitt Do you remember, is it possible that this value will be `nil` (when the value of the `DefaultAllowPrivilegeEscalation` is used)?